package.json

```diff
@@ -17,7 +17,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "cypress": "cypress open"
+    "cypress": "cypress open",
+    "sass": "sass src/Sass:src/Css --watch --no-source-map"
```

Comment:
Would enjoy hearing more about this and wonder if I need to ```npm run sass```

Reply:
Technically, that command will ensure that changes to the sass file will live update. I find that most of the time it does it on its own. But if it's not working at any time, you can run that command in a separate terminal and it should ensure that sass updates auto refresh.

Reply:
Thanks for clearing this up!